webapp/components/PasswordReset/Request.vue

```diff
@@ -124,17 +102,3 @@ export default {
   },
 }
 </script>
-<style>
-.login-image {
-  width: 90%;
-  max-width: 200px;
-}
-.password-reset {
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-09T14:13:39Z" title="Wednesday, October 9th 2019, 4:13:39 pm +02:00">Oct 9, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -124,17 +102,9 @@ export default {
   },
 }
 </script>
-<style>
+<style scoped>
 .login-image {
   width: 90%;
   max-width: 200px;
 }
```


if this is not used anymore, we should remove it, although I would like to know the motive for removing the image from these forms... they seems a bit bland

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-10T09:08:17Z" title="Thursday, October 10th 2019, 11:08:17 am +02:00">Oct 10, 2019</time>_

---


I did not really like the logo on the password reset form and would have preferred a "lock" icon or image that distinguishes the form from the signup form etc. And I think the motivation for removing it was that I had troubles to get the layout right.